README.md

```diff
@@ -20,12 +20,6 @@ them to CCFv3.
 
 ## Installation
 ### Python Version and Environment
-Note that due to some of our dependencies we're currently limited to python
-version `3.7`. Please make sure you set up a virtual environment with that
-version before trying to install this library. If you're unsure how to do that
-please have a look at [conda](https://docs.conda.io) or
-[pyenv](https://github.com/pyenv/pyenv).
-
```

Comment:
Maybe we can leave a small paragraph that mentions the python versions we support? Something like "The currently supported python versions are `3.7`, `3.8`, and `3.9`."

Reply:
Added in `setup.py` and `installation.rst` in 5cf6477.

Reply:
Thanks for updating `installation.rst` too!

I think a small word twist snuck in: it should be "currently supported python versions" instead of "currently python versions supported"

Reply:
Done in 7f2545b.

Reply:
Sorry I messed up with my instructions, should've been more precise. I fix everything in 01b23c3 🙈